src/python/analyzer-deployer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM grapl/grapl-python-build:latest AS analyzer-deployer-build
+USER grapl
+WORKDIR /home/grapl
+COPY --chown=grapl . ./analyzer-deployer
+COPY --from=grapl/grapl-python-build /home/grapl/venv venv
+RUN /bin/bash -c "source venv/bin/activate && pip install -r analyzer-deployer/requirements.txt && python -m mypy_boto3"
```

Comment:
Could you add some comments on what all this zipping maintenance is doing? (like, I can roughly figure it out, but doesn't hurt to over-comment)